<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setupForm</class>
 <widget class="QWidget" name="setupForm">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>497</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1019</width>
    <height>497</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1019</width>
    <height>497</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Kurulum Ayarları</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/resources/Weather-icon.png</normaloff>:/resources/Weather-icon.png</iconset>
  </property>
  <widget class="QLabel" name="labelTarget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>320</width>
     <height>100</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelMono">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>320</width>
     <height>100</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelEdge">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>50</y>
     <width>320</width>
     <height>100</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelHough">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>320</width>
     <height>100</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelAnalyze">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>320</width>
     <height>100</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Hedef Alan İmajı</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mono İmaj</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>30</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tespit Edilen Kenarlar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Çizgi Tespiti</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Köşe Tespiti</string>
   </property>
  </widget>
  <widget class="QPushButton" name="captureButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Yakala</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkSaveTarget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>İmaj</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkSaveMono">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>160</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>İmaj</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkSaveEdge">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>160</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>İmaj</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkSaveHough">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>İmaj</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkSaveAnalyze">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>İmaj</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkSaveMonoMatrix">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>160</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Matrix</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkSaveEdgeMatrix">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>160</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Matrix</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkSaveHoughMatrix">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>310</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Matrix</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Seçilileri Kaydet</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkSaveHoughLines">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>310</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Çizgi Matrixi</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkSaveHoughSpace">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>HSpace</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editHoughLineNo">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>200</y>
     <width>42</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>99; </string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>200</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Çizgi Sayısı</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>230</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Eşik</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="editHoughThreshold">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>230</y>
     <width>42</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>999; </string>
   </property>
   <property name="text">
    <string>200</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelVoteAvg">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>230</y>
     <width>42</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Avg</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelPrimaryLine">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>140</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>ÇİZGİ BULUNAMIYOR!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="editHoughThetaStep">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>290</y>
     <width>42</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>9.9; </string>
   </property>
   <property name="text">
    <string>1.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>290</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Açı Çöz.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="editHoughThetaMin">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>260</y>
     <width>42</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>999; </string>
   </property>
   <property name="text">
    <string>80</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>260</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Açı Aralığı</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="editHoughThetaMax">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>260</y>
     <width>42</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>999; </string>
   </property>
   <property name="text">
    <string>100</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>190</y>
     <width>321</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
    </font>
   </property>
   <property name="documentTitle">
    <string/>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="editVoidThreshold">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>320</y>
     <width>42</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>99; </string>
   </property>
   <property name="text">
    <string>30</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>320</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Void Eşiği</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="saveExitButton">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>470</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>Kaydet &amp; Çık</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>470</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>Çık</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>440</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Temizle</string>
   </property>
  </widget>
  <zorder>label_9</zorder>
  <zorder>label_7</zorder>
  <zorder>label_6</zorder>
  <zorder>label_8</zorder>
  <zorder>label_10</zorder>
  <zorder>labelTarget</zorder>
  <zorder>labelMono</zorder>
  <zorder>labelEdge</zorder>
  <zorder>labelHough</zorder>
  <zorder>labelAnalyze</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>captureButton</zorder>
  <zorder>checkSaveTarget</zorder>
  <zorder>checkSaveMono</zorder>
  <zorder>checkSaveEdge</zorder>
  <zorder>checkSaveHough</zorder>
  <zorder>checkSaveAnalyze</zorder>
  <zorder>checkSaveMonoMatrix</zorder>
  <zorder>checkSaveEdgeMatrix</zorder>
  <zorder>checkSaveHoughMatrix</zorder>
  <zorder>saveButton</zorder>
  <zorder>checkSaveHoughLines</zorder>
  <zorder>checkSaveHoughSpace</zorder>
  <zorder>labelVoteAvg</zorder>
  <zorder>labelPrimaryLine</zorder>
  <zorder>plainTextEdit</zorder>
  <zorder>saveExitButton</zorder>
  <zorder>exitButton</zorder>
  <zorder>clearButton</zorder>
  <zorder>editHoughThetaMax</zorder>
  <zorder>editHoughThetaMin</zorder>
  <zorder>editHoughThreshold</zorder>
  <zorder>editHoughThetaStep</zorder>
  <zorder>editVoidThreshold</zorder>
  <zorder>editHoughLineNo</zorder>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>captureButton</sender>
   <signal>clicked()</signal>
   <receiver>setupForm</receiver>
   <slot>captureButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>452</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>setupForm</receiver>
   <slot>saveButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>428</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>490</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>setupForm</receiver>
   <slot>clearButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>857</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>925</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>setupForm</receiver>
   <slot>exitButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>854</x>
     <y>479</y>
    </hint>
    <hint type="destinationlabel">
     <x>808</x>
     <y>474</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveExitButton</sender>
   <signal>clicked()</signal>
   <receiver>setupForm</receiver>
   <slot>saveExitButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>983</x>
     <y>481</y>
    </hint>
    <hint type="destinationlabel">
     <x>1009</x>
     <y>481</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>captureButton()</slot>
  <slot>saveButton()</slot>
  <slot>clearButton()</slot>
  <slot>saveExitButton()</slot>
  <slot>exitButton()</slot>
 </slots>
</ui>
